docker-compose.yml

```diff
@@ -28,8 +28,7 @@ services:
       ADMIN_PASSWORD: ofn123
       OFN_DB_HOST: db
     command: >
-      bash -c "(bundle check || bundle install) &&
-               wait-for-it -t 30 db:5432 &&
+      bash -c "wait-for-it -t 30 db:5432 &&
```

Comment:
I think bundle install was left there because it is fairly quick to run on every restart.